controlplane/api/v1alpha1/rke2controlplane_webhook.go

```diff
@@ -52,20 +61,48 @@ var _ webhook.Validator = &RKE2ControlPlane{}
 
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type.
 func (r *RKE2ControlPlane) ValidateCreate() error {
-	if bootstrapv1.ValidateRKE2ConfigSpec(r.Name, &r.Spec.RKE2ConfigSpec) != nil {
-		return bootstrapv1.ValidateRKE2ConfigSpec(r.Name, &r.Spec.RKE2ConfigSpec)
+	rke2controlplanelog.Info("RKE2ControlPlane validate create", "control-plane", klog.KObj(r))
```

Comment:
```suggestion
	rke2controlplanelog.Info("RKE2ControlPlane validate create", "rke2-control-plane", klog.KObj(r))
```

Reply:
should we remove this log line? It's only telling that webhook is validating a resource